EIPS/eip-crosschain-id.md

```diff
@@ -0,0 +1,131 @@
+---
+eip: <to be assigned>
+title: Crosschain Identifier Specification
+author: Weijia Zhang <weijia@wanchain.org>, Peter Robinson <peter.robinson@consensys.net>
+discussions-to:
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-10-21
+requires (*optional): <EIP number(s)>
+replaces (*optional): <EIP number(s)>
+---
+
+## Simple Summary
```

Comment:
The simple summary should really only be a couple sentences. The requirements specified here would make more sense in the motivation / rationale.

Reply:
Sure. We will review in today's meeting and make changes.